src/psd_tools/api/psd_image.py

```diff
@@ -525,6 +540,38 @@ def _get_pattern(self, pattern_id):
                         return pattern
         return None
 
+    def _clear_clipping_layers(self):
+        for layer in self.descendants():
+            layer._clip_layers = []
+            layer._has_clip_target = True
+
+    def _compute_clipping_layers(self):
```

Comment:
Appreciate if there is a short comment describing the behavior difference between tools